src/identifier.h

```diff
@@ -21,12 +21,18 @@
 
 class Identifier : public RootObject
 {
+private:
```

Comment:
Having these fields present is also a win for gdb, as I can inspect this class from both C++ and D.